examples/spark2/src/main/scala/org/apache/carbondata/examples/PreAggregateDataMapExample.scala

```diff
@@ -72,6 +73,13 @@ object PreAggregateTableExample {
          | select id,max(age) from mainTable group by id"""
         .stripMargin)
 
+    // show datamap
+    spark.sql("show datamap on table mainTable").show(false)
+
+    // drop datamap
+    spark.sql("drop datamap preagg_count on table mainTable").show()
+    spark.sql("show datamap on table mainTable").show(false)
+
     spark.sql(
       s"""
          | SELECT id,max(age)
```

Comment:
since you are adding comment, can you add for query also, to describe which datamap it will hit